Makefile

```diff
@@ -1,20 +1,22 @@
 APP ?= go-skeleton
+ENV ?= dev
+PORT ?= 8080
 
 .PHONY: build
 build:
 	@GO111MODULE=on CGO_ENABLED=0 go build -o ${APP}
 
 .PHONY: run
 run: build
-	@./${APP}
+	@bash -ac 'source .env.${ENV} && ./${APP}'
```

Comment:
it looks a little complicated for me

Reply:
Can't really find a way to do it simplier without polluting developers environment.

Reply:
okay